ietf/snippets/models.py

```diff
@@ -74,7 +74,7 @@ class WorkingGroup(models.Model, index.Indexed):
 
     @property
     def url(self):
-        return settings.DATATRACKER_URI + "/wg/" + self.acronym
```

Comment:
This should not change. Working groups are under `https://datatracker.ietf.org/wg/<acronym>`.
See https://datatracker.ietf.org/wg/

Reply:
But the ticket says it should change, no? https://github.com/ietf-tools/wagtail_website/issues/279
> the URLs built on the "Acronym field" currently take the form of https://datatracker.ietf.org/wg/FOO when the current base URL should be https://datatracker.ietf.org/group

Both versions work:
- https://datatracker.ietf.org/wg/add
- https://datatracker.ietf.org/group/add

Reply:
@mgax you are correct. And @rjsparks confirmed that `/group/<acronym>` is preferred.